lib/private/handlers/container-images.ts

```diff
@@ -208,7 +216,17 @@ class ContainerImageBuilder {
       return undefined;
     }
 
-    return (await shell(executable, { cwd: assetPath, quiet: true })).trim();
```

Comment:
Hm. This call had `quiet: true` but I don't see a value for `subprocessOutputDestination` in the call to `shell()`. What happened to it, and why doesn't it matter?